README.md

```diff
@@ -1,23 +1,30 @@
 # Pair Programming Lessons Guide/Index
 
-The main reason for this repo to exist is create an index with **topics**, **exercises** and **examples** to be used as guide and reference for people who wants to start learning programming or these whom may want to learn new things or refresh some knowledge for the freecodecamp venezuelan community.
+The main reason for this repository to exist is create a knowledge base, that
+works as guidance to people who wants to introduce themselves to programming and
+web development.
 
-We will also try to describe ideas, tools and software to make this lessons **accessible** and **achievable** for all members, taking into account that some members do not have the best hardware, internet connection or stable electric supply.
+We will also try to share tools that improves the **accessibility** for all
+members who do not have permanent internet connection or good hardware.
 
-It is well known that there are not a real path to learn programming but this guide/index aims to cover simple topics and encourage people to learn with others solving simple exercises doing pair programming or taking a part of their time learning by themselves.
+It is well known that there are not a real path to learn programming but this
+guide/index aims to cover simple topics and encourage people to learn basic
+concepts with others while solving simple exercises in our pair programming
```

Comment:
I was thinking a lot about this concept of pair programming and the people always relate it to just two folks doing an activity about programming or software development.

Maybe we can change it later or add the word workshop, it would be like workshop/pair programming

I don't what do you think @Hmerin @Gracrys

Reply:
We can add this description right away, because past activities were more workshops than pair programming Ex.:

1. Small intro to the topic
2. Divide people in pairs to solving the same problem/exercise
3. Discuss different approaches.

We haven’t think yet about a methodology to organise the activity part, and today it is very improvised in many aspects so we can call it whatever it fits better.